quality-metrics/new/qm-000-integration-tests.md

```diff
@@ -0,0 +1,32 @@
+---
+SMQM: UNASSIGNED
+Author: JJ Asghar <jj@chefio>
+Status: Draft
+License: Apache 2.0
+---
+
+# Integration Testing
+
+There should be at least one of the following with integration testing:
+* serverspec
+* bats
+* TBD
+
+With integration testing you can verify via test-kitchen that the cookbook does what you expect it to do, without checking it by hand.
+
+Integration testing is pinical to velocity and verifing community cookbooks are what we expect them to be.
+
+### Verification
+
+Pseudocode or actual code that can be used to automatically verify the rule and/or assign appropriate points.
+
+  find test/integration/default/ --name *_spec.rb > 3
```

Comment:
I think outside of executing the tests we should just be checking to see if they exist